packages/theme/src/theme/default.ts

```diff
@@ -1,149 +1,146 @@
 import { createStitches } from "@stitches/react";
 
-export const { styled, css, theme, keyframes, getCssText } = createStitches({
-  prefix: "pluto",
-  theme: {
-    borderWidths: {
-      borderWidth0: "0px",
-      borderWidth10: "1px",
-      borderWidth20: "2px",
+export const { styled, css, theme, keyframes, getCssText, config } =
+  createStitches({
+    prefix: "pluto",
+    theme: {
+      borderWidths: {
+        borderWidth0: "0px",
+        borderWidth10: "1px",
+        borderWidth20: "2px",
+      },
+      colors: {
+        colorTextStrongest: "#0F172A",
+        colorTextStronger: "#334155",
+        colorText: "#64748B",
+        colorTextError: "#B91C1C",
+        colorTextWarning: "#B45309",
+        colorTextSuccess: "#047857",
+        colorTextInverse: "#FFFFFF",
+        colorTextLink: "#102EE9",
+        colorTextLinkStrong: "#0B1F9C",
+        colorTextHeading: "#041162",
+        colorBackground: "#FFFFFF",
+        colorBackgroundWeakest: "#F8FAFC",
+        colorBackgroundWeaker: "#F7F9FF",
+        colorBackgroundWeak: "#F1F5F9",
+        colorBackgroundStrong: "#CBD5E1",
+        colorBackgroundPrimary: "#102EE9",
+        colorBackgroundPrimaryStrong: "#0B1F9C",
+        colorBackgroundPrimaryStrongest: "#041162",
+        colorBackgroundDestructive: "#DC2626",
+        colorBackgroundDestructiveStrong: "#B91C1C",
+        colorBackgroundInfo: "#EBEDFF",
+        colorBackgroundSuccess: "#ECFDF5",
+        colorBackgroundWarning: "#FFFBEB",
+        colorBackgroundError: "#FEF2F2",
+        colorBackgroundBody: "#F7F9FF",
+        colorBackgroundGradientTasks:
+          "linear-gradient(177.5deg, #FF9788 -9.4%, #3440DF 64.04%, #102EE9 97.66%)",
+        colorBackgroundGradientProcesses:
+          "linear-gradient(175.85deg, #9FFED5 4.23%, #3767E4 53.02%, #234AE6 70.91%, #102EE9 96.37%)",
+        colorBorderWeakest: "#E2E8F0",
+        colorBorderWeaker: "#CBD5E1",
+        colorBorder: "#64748B",
+        colorBorderStrongest: "#0F172A",
+        colorBorderError: "#EF4444",
+        colorBorderWarning: "#D97706",
+        colorBorderSuccess: "#059669",
+        colorBorderPrimary: "#102EE9",
+        colorIconError: "#DC2626",
+        colorIconWarning: "#D97706",
+        colorIconSuccess: "#059669",
+        colorIconInfo: "#102EE9",
+        colorIconWeaker: "#94A3B8",
+        colorIconWeak: "#64748B",
+        colorIconStrong: "#334155",
+        colorIconStronger: "#0F172A",
+      },
+      fontSizes: {
+        fontSize10: "0.75rem", // 12px
+        fontSize20: "0.875rem", // 14px
+        fontSize30: "1rem", // 16px
+        fontSize40: "1.25rem", // 20px
+        fontSize50: "1.5rem", // 24px
+        fontSize60: "2rem", // 32px
+        fontSize70: "2.5rem", // 40px
+        fontSize80: "3.375rem", // 54px
```

Comment:
Will Figma allow showing the font size as `fontSize10` when inspecting a mock up? If not, I think it gets a bit annoying to translate pixels/rems to the right font size name.

Reply:
**Short answer:** Yes.

**Slightly longer answer:** It's going to take some setup on the Figma side to accomplish that. As the design team works through creating symbols and components with the new tokens, there are some plugin options that will display that info in the inspect panel. I'll be working with them to figure out the best option.

**Side note:** I totally understand that using a scale token naming scheme is difficult to translate at the beginning. As the system gets used more, that part gets easier. The naming allows us to scale as well. Using a scale naming convention disconnects the token name from the actual value, allowing the value to be anything. So `fontSize30` can be `16px` now, but in the future it could be `18px`. In order to implement that change, we'd only have to change the token value and not the token uses in each component.

Reply:
Awesome, makes sense!